frontend/src/app/application-forms/temporary-outfitters/temporary-outfitters.component.ts

```diff
@@ -350,7 +374,9 @@ export class TemporaryOutfittersComponent implements DoCheck, OnInit {
   onSubmit() {
     this.submitted = true;
     this.numberOfFilesToUpload();
+    console.log(this.numberOfFilesToUpload());
     this.checkFileUploadValidity();
+    console.log(this.checkFileUploadValidity());
```

Comment:
let's get rid of these extra console log messages

Reply:
@briandavidson - Removed